cmake/ArborXConfig.cmake.in

```diff
@@ -5,8 +5,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR} ${CMAKE_MODULE_PATH})
 include(CMakeFindDependencyMacro)
 find_dependency(Kokkos)
 
-include("${CMAKE_CURRENT_LIST_DIR}/ArborXSettings.cmake")
-if(ARBORX_ENABLE_MPI)
+if(@ARBORX_ENABLE_MPI@)
```

Comment:
```suggestion
set(ArborX_ENABLE_MPI @ARBORX_ENABLE_MPI@)
if(ArborX_ENABLE_MPI)
```
The changes you propose lose the ability to tell at configuration time when finding ArborX whether it was built with or without MPI.  Note that I spelled the variable in pascal case on purpose.  It makes me think we should consider renaming our target to `ArborX::arborx` (I am not asking to do this here)

Reply:
Agree with the code. But I am sure the naming will become confusing which case to use when, and prone to errors. While I do not have any workarounds at the moment, I would recommend trying really hard to avoid that. Barring that, we would need to document exactly when to use each case. So, in the current form, this PR should come with an update in developer docs.

Reply:
Why do you want to use a different case for the original `CMake` variable and the one here?